doc/source/conf.py

```diff
@@ -0,0 +1,204 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
```

Comment:
Did you copy this comment block from Sphinx template? The word `does` here seem awkward (although the grammar is correct).

Reply:
yeah, I copied the header from the sphinx template. I will edit it though. you are right.